airbyte-workers/src/main/java/io/airbyte/workers/temporal/TemporalClient.java

```diff
@@ -213,6 +265,11 @@ void refreshRunningWorkflow() {
     } while (token != null && token.size() > 0);
   }
 
+  public Set<String> getAllRunningWorkflows() {
```

Comment:
The migrator app calls this to list all open workflows in either instance

Reply:
👍  Added javadoc. I'll add some time measurement logging to get a feel for performance. I'm not sure how we'd identify quarantined workflows by tweaking this, as I suspect we'd need an entirely different approach for querying temporal compared to the existing `listOpenWorkflowExecutions` that we call for listing running workflows. @benmoriceau  might have a better idea of how to answer your question here

Reply:
In order to find all quarantined workflows, I believe we would need to call the `getQuarantinedInformation()` query method on every running workflow. I'm not exactly sure what the performance impact of this would be, but I imagine it would make it significantly slower. We also may run into rate limiting issues (at least we did when trying to do something similar with `tctl` - maybe it is different when calling the API directly?)